File: JAVA/02_If/BOJ_05_2884_알람시계.java
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringTokenizer st = new StringTokenizer(br.readLine()," ");
		int hour = Integer.parseInt(st.nextToken());
		int minute = Integer.parseInt(st.nextToken());
        if (0 <= hour && hour <= 23 && 0 <= minute && minute <= 59) {
            if (hour == 0 && minute < 45) {
                System.out.println(23 + " " + (minute + 15));
            }
            else if (minute >= 45) {
                System.out.println(hour + " " + (minute - 45));
            }
            else if (hour != 0 && minute < 45) {
                System.out.println((hour - 1) + " " + (minute + 15));
            }
        }
        else System.out.println(0);
    }
}

File: C/02_If/BOJ_05_2884_알람시계.c
#include <stdio.h>
main() {
	int hour, minute;
	scanf("%d %d", &hour, &minute);
	if (hour == 0 && minute < 45) printf("23 %d", minute + 15);
	else if (minute >= 45) printf("%d %d", hour, minute - 45);
	else if (hour != 0 && minute < 45) printf("%d %d", hour - 1, minute + 15);
}